docs/source/compile/get-started.rst

```diff
@@ -159,6 +159,6 @@ Here is a few examples of existing methods and their limitations:
 -  ``torch._dynamo`` works out of the box and produces partial graphs.
    It still has the option of producing a single graph with
    ``nopython=True`` which are needed for `some
-   situations <./documentation/FAQ.md#do-i-still-need-to-export-whole-graphs>`__
+   situations <https://pytorch.org/docs/stable/dynamo/faq.html#do-i-still-need-to-export-whole-graphs>`__
```

Comment:
```suggestion
   situations <https://pytorch.org/docs/stable/compile/faq.html#do-i-still-need-to-export-whole-graphs>`__
```

Reply:
We recently removed the `dynamo` folder for docs

Reply:
Thanks for clearing it out to me!
Whenever I paste the link with ```compile``` in the browser it gets back to the same "dead end" page. Is that the expected behavior ?